app/controllers/import_controller.rb

```diff
@@ -106,6 +106,21 @@ def import
               @errors << ValidationError.new(error, row_ind).message
             end
           end
+
+          vaccines = []
+          vaccines.push(vaccination(row[102..106], row_ind)) if row[102..106].filter(&:present?).any?
+          vaccines.push(vaccination(row[107..111], row_ind)) if row[107..111].filter(&:present?).any?
+          patient[:vaccines_attributes] = vaccines unless vaccines.empty?
+
+          # Validate using Vaccine model validators without saving
+          vaccines.each do |vaccine_data|
+            validation_vaccine = Vaccine.new(vaccine_data)
+            next if validation_vaccine.valid?(:import)
```

Comment:
Very minor, but I don't think we actually need the `:import` context here. When the vaccine model was added, we learned from the past and enabled the validations for the UI from day 1, so there are no individual validation contexts on the vaccine model.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/958/commits/ab7f19cf8ad4bd93f4d65ff43baf58e9f4c9c392#diff-ee31e076a80cc9f48da0394c0cb066d59bf2f8c68f68eb62cb5119d01712d90eR118